### ISSUE TITLE
Adjusted Unit Tests 4 3 Calculation Functions (Assumed testing is not needed for input or output functions)

### ISSUE THREAD

**Issue by benrandoing20** [benrandoing20/ClassworkBME547#14]
https://github.com/benrandoing20/ClassworkBME547/blob/a7d52515123863865babe6a9605a368d3c77c2c4/test_line_yvalue.py#L1

**Comment by dward2** [comment]
Correct, you do not have to write tests for functions that only do input/output.  Again, the need for modular code.

I have seen all your issues.  Thanks.
